<ui version="4.0" >
 <class>MainDialog</class>
 <widget class="QWidget" name="MainDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>mainDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="m_fileSrcButtonGroup" >
     <property name="title" >
      <string/>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QRadioButton" name="m_allFilesButton" >
        <property name="text" >
         <string>Show all images in current al&amp;bum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_selectedFilesButton" >
        <property name="text" >
         <string>Show onl&amp;y selected images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_customButton" >
        <property name="text" >
         <string>Cu&amp;stom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_ImagesFilesGroup" >
     <property name="whatsThis" >
      <string>Preview the currently selected image.</string>
     </property>
     <property name="title" >
      <string>Image Files in slideshow</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item rowspan="6" row="0" column="0" >
       <widget class="KIPISlideShowPlugin::ListImageItems" name="m_ImagesFilesListBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>This is the list of the image files for your portfolio.
The portfolio's first image is on the top; the last image is on the bottom.
If you want to add some images, click on the 'Add' button or use the drag-and-drop.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer name="spacer14" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>71</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item rowspan="6" row="0" column="2" >
       <widget class="KIPISlideShowPlugin::ImageViewer" native="1" name="m_ImageViewer" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="m_ImagesFilesButtonUp" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis" >
         <string>Moving the current image up on the portfolio list.</string>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="m_ImagesFilesButtonAdd" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis" >
         <string>Add some image files to the portfolio list.</string>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QPushButton" name="m_ImagesFilesButtonDelete" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis" >
         <string>Remove some image files from the portfolio list.</string>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QPushButton" name="m_ImagesFilesButtonDown" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis" >
         <string>Moving the current image down on the portfolio list.</string>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <spacer name="spacer14_2" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>71</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="m_label6" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>Total number of images in the portfolio and sequence duration.</string>
        </property>
        <property name="text" >
         <string>None</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2" >
       <widget class="QLabel" name="m_label7" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>Currently selected image in the portfolio list.</string>
        </property>
        <property name="text" >
         <string>None</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="1" column="0" >
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Content</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QCheckBox" name="m_printNameCheckBox" >
            <property name="text" >
             <string>P&amp;rint filename</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_printProgressCheckBox" >
            <property name="text" >
             <string>Progress indicator</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="m_printCommentsCheckBox" >
          <property name="text" >
           <string>Pr&amp;int captions </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Video</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <widget class="QCheckBox" name="m_openglCheckBox" >
          <property name="text" >
           <string>Use Open&amp;GL slideshow transitions</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>14</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item rowspan="2" row="0" column="1" >
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Playback</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5" >
        <item>
         <widget class="QCheckBox" name="m_loopCheckBox" >
          <property name="text" >
           <string>&amp;Loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_shuffleCheckBox" >
          <property name="text" >
           <string>&amp;Shuffle images</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" colspan="2" >
           <widget class="QLabel" name="m_delayLabel" >
            <property name="text" >
             <string>Delay between images (s):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QSpinBox" name="m_delaySpinBox" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="value" >
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Transition effect:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2" >
           <widget class="QComboBox" name="m_effectsComboBox" />
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>44</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIPISlideShowPlugin::ListImageItems</class>
   <extends>QListView</extends>
   <header>listimageitems.h</header>
  </customwidget>
  <customwidget>
   <class>KIPISlideShowPlugin::ImageViewer</class>
   <extends>QWidget</extends>
   <header>imageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
